models/service.js

```diff
@@ -20,57 +14,64 @@ service.init(
       autoIncrement: true,
     },
 
-    //title of service 
+    //title of service
     service_title: {
       type: DataTypes.STRING,
       allowNull: false,
     },
-    //availability
-    availability: {
-        type: DataTypes.STRING,
-        allowNull: false,
+
+    //service start
+    service_start: {
+      type: DataTypes.STRING,
```

Comment:
Please make this a datetime datatype

Reply:
Please also add some validation to these to make sure they are dates: https://sequelize.org/docs/v6/core-concepts/validations-and-constraints/#validators

Reply:
done